eng/pipelines/installer/jobs/build-job.yml

```diff
@@ -1,384 +0,0 @@
-parameters:
```

Comment:
Here's a list of other files that look to be merge leftover I made with `git diff 9b57a265c7e --stat=200 --diff-filter A`:
```
 eng/pipelines/installer/jobs/build-job.yml                                                                                   |  384 ++++++
 eng/pipelines/runtimelab/coreclr-tests.yml                                                                                   |   41 +
 src/coreclr/Corehost.Static/config.h                                                                                         |    7 +
 src/coreclr/inc/clr_std/utility                                                                                              |  269 ++++
 src/coreclr/nativeaot/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/CallConverterThunk.cs                        | 1414 ++++++++++++++++++++
 src/coreclr/tools/Common/JitInterface/CorInfoImpl.Intrinsics.cs                                                              |  156 +++
 src/coreclr/tools/aot/ILCompiler.LLVM/libLLVMdep.depproj                                                                     |   26 +
 src/libraries/Native/Unix/CMakeLists.txt
 src/libraries/System.Linq.Expressions/src/MatchingRefApiCompatBaseline.txt                                                   |   21 +
 src/coreclr/System.Private.CoreLib/src/System/Exception.CoreCLR.LLVM.cs                                                      |   42 +
```

Reply:
Nice!